### PR TITLE
Control loading of the zlibNX library on AIX with -XX:[+|-]UseZlibNX

### LINKED ISSUE #1178: Control the behavior of loading the zlibNX library on AIX via -XX:[+/-]UseZlibNX
**Issue or pull request number:**
https://github.com/ibmruntimes/openj9-openjdk-jdk17/pull/265
https://github.com/eclipse-openj9/openj9/pull/18164

**Overview:**
Added the option -XX:[+/-]UseZlibNX to control the behavior of loading the zlibNX library on AIX, The original behavior is it gets loaded on P9 or later when it's available on the system, and this part doesn't change.

**Release target:**
Eclipse OpenJ9 0.41

**Applies to the following JDK versions:**
All

**Applies to the following platforms:**
AIX

**For new command line options:**

-XX:[+/-]UseZlibNX

- Purpose: On P9 or later when zlibNX is available, the zlibNX directory will only be added to the LIBPATH if -XX:-UseZlibNX is not specified
- Restrictions: None

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-docs/issues/1178

Created a new topic for -XX:[+|-]UseZlibNX. Updated the What's new in version 0.41.0

Closes #1178
Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

### CODE REVIEW

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have updated the documents. Please check and confirm. Thanks!

File: docs/xxusezlibnx.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+| Setting               | Effect  | Default                                                                            |
+|-----------------------|---------|:----------------------------------------------------------------------------------:|
+| `-XX:+UseZlibNX` |  Enable  |  :fontawesome-solid-check:{: .yes aria-hidden="true"}<span class="sr-only">yes</span>         |
+| `-XX:-UseZlibNX` | Disable |     |
+
+
+## Explanation
+
+AIX system adds the `zlibNX` library location, if available, to the `LIBPATH` variable by default. But having the `zlibNX` library directory location in the `LIBPATH` variable might cause some issues, such as Git clone failing when run from Java&trade;.
```
Comment (pshipton) [inline docs/xxusezlibnx.md:43]: such as Git clone failing when run from Java
->
such as Git clone failing when exec'ed from Java when `zlibNX` is on the LIBPATH in the environment.

Comment (Sreekala-Gopakumar) [reply docs/xxusezlibnx.md:43]: what does "exec'ed" mean? executed?

Comment (pshipton) [reply docs/xxusezlibnx.md:43]: Yes, I'm referring to a Java API such as https://docs.oracle.com/javase/8/docs/api/java/lang/Runtime.html#exec-java.lang.String:A-java.lang.String:A-java.io.File-

**Comment by Sreekala-Gopakumar** [comment]
Jenkins doc stage

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have updated the document. Please check and confirm. Thanks!

**Review by pshipton — APPROVED** [review]

**Pull request merged**